subsys/emds/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+zephyr_sources(emds.c)
```

Comment:
I guess it is better to use `zephyr_sources_ifdef` and dependency on the configuration option EMDS

Reply:
My plan was to put that in the CMakeLists.txt file that takes the hole folder. So if EMDS is not added this file is not added.